bigflow/build/dataflow/workerdeps/beam2.24_py3.5.txt

```diff
@@ -0,0 +1,111 @@
+# list of preinstalled dependencies on Dataflow workers
```

Comment:
Is it possible that one of the dependencies gets a patch that list will be obsolete (so the user will constantly not match these requirements) ?

Reply:
I wasn't able to find a direct answer in the documentation.
However old "beam" versions (like 2.25) still uses "old" libraries (even don't get minor updates).
So IMO it is safe to assume that "versions are newer change". If they are - we will recognise this via 'git diff' after rerunning `grab_beam_worker_deps.py`.